portfolio/src/main/webapp/script.js

```diff
@@ -56,6 +56,13 @@ function loadSkills () {
  */
  async function fetchFromData() {
   const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('secret-message').innerText = quote; 
+  // This gives a list of messages
+  const msgs = await response.json();
+  console.log(msgs); 
+
+  // Adds all of the messages to an unordered list
+  msgs.forEach(m => {
```

Comment:
Would you need to clear $("#all-messages") first? What happens if you click the button twice?

Reply:
that's smart!! ok on it